satpy/readers/utils.py

```diff
@@ -476,6 +477,48 @@ def remove_earthsun_distance_correction(reflectance, utc_date=None):
     return reflectance
 
 
+def get_serializable_dask_array(manager, varname, chunks, dtype):
+    """Construct a serializable dask array from a variable.
+
+    When we construct a dask array using da.array and use that to create an
```

Comment:
Should we specify that this happens when a dask array is constructed from a file?

Reply:
Yes.  Fixed in 5b137e8e7.